tools/build_chain.sh

```diff
@@ -1286,6 +1294,66 @@ check_bin()
     echo "Binary check passed."
 }
 
+prepare_ca(){
+
+    if [ -z ${CertConfig} ] || [ ! -e ${CertConfig} ];then
+        # CertConfig="${output_dir}/cert.cnf"
+        generate_cert_conf "cert.cnf"
+    else 
+        cp ${CertConfig} .
+    fi  
+    dir_must_not_exists "${output_dir}/chain"
+    if [ -z "$ca_path" ]; then
+        echo "Generating CA key..."
+        gen_chain_cert ${output_dir}/chain >"${logfile}" 2>&1 || exit_with_clean "openssl error!"
+        mv ${output_dir}/chain ${output_dir}/cert
+    else
+        echo "Use user specified ca key from ${ca_path}"
+        mkdir -p ${output_dir}/cert
+        cp "${ca_key}" ${output_dir}/cert
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5331982)